### ISSUE TITLE
Play gifs completely

### ISSUE THREAD

**Issue by ififfy** [ififfy/flipflip#50]
We already include a GIF-reading JS library, which should be able to tell us how long a gif is.

The tricky part is actually the data flow: getting `ImagePlayer` to tell its parent `HeadlessScenePlayer` how long the gif is, and override the next-scene timeout in the right spot.

**Comment by GoddessServant0** [comment]
> We already include a GIF-reading JS library

Which library are you referring to? `animated-gif-detector`?  (I was sadly disappointed by its terrible performance, have been looking at an alternative for truly detecting gifs, but very quickly)

**Comment by GoddessServant0** [comment]
So I grew tired of `animated-gif-detector`'s poor performance and limited usability. Found an alternative library which seems to perform better anyway: https://github.com/Prinzhorn/gif-info

This was implemented in #53 

I stored the duration in a variable, but didn't do anything with that yet. Not exactly sure how to bubble it up. Will look into, but could use your assistance since you're more familiar with the Player design
